RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -185,6 +259,20 @@ private string CreateBaseWorkingDirectory()
             return baseDirectory;
         }
 
+        private bool HasAnalyzers(IPackage package, string language)
+        {
+            return GetAnalyzers(package, language).Any();
+        }
+
+        /// <summary>
+        /// Retrieves the analyzers contained within a given NuGet package corresponding to a given language
+        /// </summary>
+        private IEnumerable<DiagnosticAnalyzer> GetAnalyzers(IPackage package, string language)
+        {
+            string packageDir = this.packageHandler.GetLocalPackageRootDirectory(package);
+            return GetAnalyzers(packageDir, this.packageHandler.LocalCacheRoot, language);
+        }
```

Comment:
Actually, all usages have been changed to point to the new one. Will merge them.

Accepted, in progress.